store/poi.go

```diff
@@ -70,16 +76,21 @@ func (m *mongoCommunityStore) GetPOISummarizedRatings(ctx context.Context, poiID
 			}),
 			AggregationUnwind("$rating"),
 			AggregationGroup(bson.M{
-				"k":  "$rating.k",
-				"id": "$id",
+				"k":            "$rating.k",
+				"id":           "$id",
+				"last_updated": "$timestamp",
```

Comment:
Aggregation on timestamp (also use `$max`) can already happen at this stage.

Reply:
I can not use `bson.E{"last_updated", "$_id.last_updated"),` in next group. It will throw error `"The field 'last_updated' must be an accumulator object"` unless I move it into _id field. But I want it be place in the first level.

Reply:
Besides, even it can happen in the first stage, is there any special reason that it must happen at the first one?